.travis.yml

```diff
@@ -1,7 +1,7 @@
 language: bash
 
 script:
-  - /bin/true
+  - echo YOLO
```

Comment:
Um excuse me.

`- echo "CHEF IS THE BEST SYSTEM ON THE PLANET"`

You forgot that one.